include/cantera/thermo/Phase.h

```diff
@@ -276,6 +281,26 @@ class Phase
 
     //!@} end group Element and Species Information
 
+    //! Return whether phase represents a stoichiometric (fixed composition)
+    //! substance
+    virtual bool isPure() const {
+        return false;
+    }
+
+    //! Return whether phase represents a compressible substance
+    virtual bool isCompressible() const {
+        return true;
+    }
+
+    //! Return a vector with default properties defining the state of a phase
+    virtual std::vector<std::string> defaultState() const;
```

Comment:
"default" doesn't feel like the right name for this to me. I would say "intrinsic", except that already has a meaning in thermo. Maybe "native"?

Reply:
`nativeState` sounds good. Adopted.